packages/clients/meldemichel/src/mapConfigurations.ts

```diff
@@ -248,11 +248,13 @@ const mapConfigurations: Record<
   }),
   [MODE.SINGLE]: () => ({
     ...commonMapConfiguration,
+    addressSearch: { ...addressSearch, displayComponent: false },
```

Comment:
I found this to be confusing with the `displayComponent: true` above and would favour the statement in line 30 to be `mode !== MODE.SINGLE` so we later have less positions to consider for behaviour understanding/change.